lsp-mode.el

```diff
@@ -2630,7 +2630,9 @@ an Elisp regexp."
      "---"
      ("Debug"
       :active (bound-and-true-p dap-ui-mode)
-      :filter ,(lambda (_) (nthcdr 3 dap-ui-menu-items)))))
+      :filter ,(lambda (_)
+                 (and (boundp 'dap-ui-menu-items)
+                      (nthcdr 3 dap-ui-menu-items))))))
```

Comment:
You may want to use `if (boundp ...) (...) "Not available")` instead, when I tested with
``` elisp
(x-popup-menu `((10 10) ,(get-buffer-window))
              (easy-menu-create-menu
               nil
               `(("Test"
                  :active (bound-and-true-p x)
                  :filter ,(lambda (_) (if (boundp 'y) (nthcdr 3 y) "Not avail")))
                 ("Test-B" :active (not (bound-and-true-p y))
                  ["Test-b" execute-extended-command]))))
```
not having the `if` will make the item `bold` instead of grey-out.